core/lib/spree/event/subscriber.rb

```diff
@@ -66,9 +73,11 @@ def event_action(method_name, event_name: nil)
       def subscribe!
         unsubscribe!
         event_actions.each do |event_action, event_name|
-          send "#{event_action}_handler=", Spree::Event.subscribe(event_name) { |event|
-            send event_action, event
-          }
+          subscription = Spree::Event.subscribe(event_name) { |event| send event_action, event }
+          # old mappings, to be removed when Solidus 2.10 is not supported anymore:
```

Comment:
Can we use the word deprecated instead of `old`? Just because it will be easier to find it if we perform a search of all the deprecated code within the codebase with a text editor.